ecosystem/typescript/sdk/src/aptos_types/type_tag.ts

```diff
@@ -190,17 +190,16 @@ export class StructTag {
    * @returns
    */
   static fromString(structTag: string): StructTag {
-    // Type args are not supported in string literal
-    if (structTag.includes("<")) {
-      throw new Error("Not implemented");
-    }
-
-    const parts = structTag.split("::");
-    if (parts.length !== 3) {
```

Comment:
why remove this check..?

Reply:
Good question!

We actually want to allow nested struct tag here, so the `length !== 3` check would definitely fail if the input is a nested struct tag. e.g: `0x1::coin::Coin<0x2::coin::MyCoin, 0x1::module::Struct<0x2::coin::MyCoin>>`